src/tiledimage.js

```diff
@@ -849,8 +849,8 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
     },
 
     /**
-     * @private
      * Get the point around which this tiled image is rotated
+     * @private
```

Comment:
JSDoc complained about having the @private above the description, so I moved it down, FYI.